### ISSUE TITLE
Question regarding to training CelebAHQ_128x128

### ISSUE THREAD

**Issue by ChenChengKuan** [yilundu/comet#2]
Hello
Thanks for your great work. I have question about line [360-387 ](https://github.com/yilundu/comet/blob/299fe629d4cf33894a18f1b3786b92737ca5199c/train.py#L360) in train.py. If we run the command in the doc directly, line 387 causes error because vgg_loss is not used when the iteration is less than 10000. 

```
            if it < 10000 or FLAGS.dataset != "celebahq_128":
                loss = im_loss
            else:
                vgg_loss = loss_fn_vgg(im_negs[:, -1], im).mean()
                loss = vgg_loss  + 0.1 * im_loss

            loss.backward()
            if FLAGS.gpus > 1:
                average_gradients(models)

            [torch.nn.utils.clip_grad_norm_(model.parameters(), 10.0) for model in models]
            [optimizer.step() for optimizer in optimizers]
            [optimizer.zero_grad() for optimizer in optimizers]

            if it % FLAGS.log_interval == 0 and rank_idx == 0:
                loss = loss.item()
                energy_pos_mean = energy_pos.mean().item()
                energy_neg_mean = energy_neg.mean().item()
                energy_pos_std = energy_pos.std().item()
                energy_neg_std = energy_neg.std().item()

                kvs = {}
                kvs['loss'] = loss
                kvs['ml_loss'] = ml_loss.item()
                kvs['im_loss'] = im_loss.item()

                if FLAGS.dataset == "celebahq_128":
                    kvs['vgg_loss'] = vgg_loss.item()
```
Another question is that why do we add vgg_loss to im_loss when the iteration pass 10000? Is there any intuition or purpose behind this design? (Can we use vgg loss directly at the very beginning?) Thanks

**Comment by yilundu** [comment]
Hi,
Thanks for the comments -- I've have pushed code to remove the error at line 387 (by not logging 'vgg_loss' when the dataset is 'celebahq_128').

I found that you are unfortunately unable to use vgg loss at the beginning because it encourages the model to ignore reconstruction of the image. Adding vgg loss later sharpens the generations of the image, with the choice of 10000 iterations a bit arbitrary.

**Comment by ChenChengKuan** [comment]
Hi,
I see. Thanks for clarifications!
